scripts/_download.js

```diff
@@ -19,17 +22,21 @@ const GITHUB_API = "https://api.github.com";
  * @returns {object}
  */
 export async function getGitHubReleaseMetadata(org, repo, releaseTag) {
-  const url = `${GITHUB_API}/repos/${org}/${repo}/releases/tags/${releaseTag}`;
-  console.log(
-    `Fetching GitHub release metadata for ${yellow(`${org}/${repo}`)} release: ${yellow(releaseTag)}`,
-  );
-
-  const response = await fetch(url);
-  if (!response.ok) {
-    throw new Error(`Failed to fetch release metadata: ${response.statusText}`);
-  }
+  try {
+    const url = `${GITHUB_API}/repos/${org}/${repo}/releases/tags/${releaseTag}`;
+    console.log(
+      `Fetching GitHub release metadata for ${yellow(`${org}/${repo}`)} release: ${yellow(releaseTag)}`,
+    );
+
+    const response = await fetch(url);
+    if (!response.ok) {
+      throw new Error(`Failed to fetch release metadata: ${response.statusText}`);
+    }
 
-  return await response.json();
+    return await response.json();
+  } catch (error) {
+    throw new Error(`Error in getGitHubReleaseMetadata: ${error.message}`);
+  }
```

Comment:
I'm not sure about the try/catches like this.  I need to look more at how this all flows.  A failed await is equivalent to a Promise.reject and should bubble up w/o needing the Errors.

I need to verify my should.

Reply:
removed these as it belongs to https://github.com/konveyor/editor-extensions/issues/235